pkgs/development/python-modules/google_cloud_org_policy/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder, google_api_core }:
+
+buildPythonPackage rec {
+  pname = "google-cloud-org-policy";
+  version = "0.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0ncgcnbvmgqph54yh2pjx2hh82gnkhsrw5yirp4wlf7jclh6j9xh";
+  };
+
+  disabled = pythonOlder "3.5";
+
+  propagatedBuildInputs = [ google_api_core ];
+
+  doCheck = false;
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

Reply:
Unless I am missing it the repo has no tests and the next line is already using `pythonImportsCheck`.

Reply:
upstream does have tests https://github.com/googleapis/python-asset

Reply:
@jonringer I think maybe you're getting confused since this thread is on an outdated bit of code. The highlighted `doCheck = false` is on a different package than the one you linked to. You linked to `python-asset`, which corresponds to the `google_cloud_asset` package in this PR, and that has been running the tests.

I am going to resolve this thread to prevent confusion. Please take another look when you get a chance.